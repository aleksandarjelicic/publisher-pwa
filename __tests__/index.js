import Preact from 'preact'
import { render } from '@testing-library/preact'
import Index from '../pages/index'

describe("ErrorLog/ErrorLog", () => {
  it("renders correctly", () => {
    const { container } = render(<Index />);

    expect(container.firstChild).toMatchSnapshot();
  })
})