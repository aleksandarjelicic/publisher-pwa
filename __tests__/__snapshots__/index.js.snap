// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLog/ErrorLog renders correctly 1`] = `
<div
  __source="[object Object]"
>
  <div
    __source="[object Object]"
  >
    <ul
      __source="[object Object]"
      class="nav"
    />
  </div>
  <main
    __source="[object Object]"
  >
    this is index
  </main>
  <div
    __source="[object Object]"
  >
    <ul
      __source="[object Object]"
      class="nav"
    />
  </div>
</div>
`;
